.cirrus.yml

```diff
@@ -28,6 +29,8 @@ env:
     FEDORA_PRIOR_IMAGE_NAME: "fedora-cloud-base-gcp-35-1-2-x86-64"
     UBUNTU_IMAGE_NAME: "ubuntu-2204-jammy-v20220622"
     UBUNTU_PRIOR_IMAGE_NAME: "ubuntu-2004-focal-v20220701"
+    UBUNTU_PRIOR2_IMAGE_NAME: "ubuntu-1804-bionic-v20220901"
+    UBUNTU_SNAP_IMAGE_NAME: "ubuntu-2204-jammy-v20220622"
```

Comment:
Is there a reason to not just use the 'regular' `UBUNTU_IMAGE_*` fields for the snap build? If that image can/should be updated for either/both the snap build and the actual tests, we can do that just as easily. For example, I've started reviewing the images for Fedora/Centos every 3 months (speaking of which...it's about that time for me) where we'll pick up the newer images for those releases.

Reply:
I could, but the SNAP is based on `core22`, i.e. 22.04, and will stay that way until there are dependency issues. I could change to UBUNTU_IMAGE_NAME, but will change in 2024, when the next LTS release comes, and hence the SNAP build would change to `UBUNTU_PRIOR_IMAGE_NAME`. So, was looking to future proof that, so that it will stay consistent.

Now, when the we decide to update the `snapcraft.yaml` to use `core24`, which is unlikely (unless there is a major change for some package to be required, such as magic this time round), that variable won't change for a while.

I hope that makes sense

Reply:
Ah, ok - yep, that tracks.